proto/api.proto

```diff
@@ -721,10 +721,11 @@ message GetAccountBalanceResponse {
 
 message TokenBalance {
   string symbol = 1;
-  string address = 2;
-  double walletAmount = 3;
-  double unsettledAmount = 4;
-  double openOrdersAmount = 5;
+  string tokenAddress = 2;
+  string payerAddress = 3;
```

Comment:
`payerAddress` isn't really the right term for this. I think `walletAddress` is what you want. People will want to do different things with their wallet address besides place Serum/Openbook orders, for which `payerAddress` is the relevant term